packages/web/src/brandkit/IconsPage.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+import { withNamespaces } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+
+import { brandStyles, GAP } from 'src/brandkit/common/constants'
+import Page from 'src/brandkit/common/Page'
+import PageHeadline from 'src/brandkit/common/PageHeadline'
+import { I18nProps, NameSpaces } from 'src/i18n'
+
+import Fetch from 'src/brandkit/common/Fetch'
+import IconShowcase from 'src/brandkit/common/Showcase'
+
+export default React.memo(function IconsPage() {
+  return (
+    <Page
+      sections={[
+        {
+          id: 'overview',
+          children: <Overview />,
+        },
+      ]}
+    />
+  )
+})
+
+interface IconData {
+  description: string
+  name: string
+  preview: string
+  uri: string
+}
+
+const LOADING = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
```

Comment:
if you want to be fancy: https://stackoverflow.com/questions/3746725/how-to-create-an-array-containing-1-n